velox/functions/sparksql/tests/SliceTest.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/lib/Slice.h"
+#include "velox/functions/lib/tests/SliceTestBase.h"
+
+namespace facebook::velox::functions::sparksql::test {
+
+namespace {
+class SliceTest : public SliceTestBase {
+ protected:
+  void SetUp() override {
+    FunctionBaseTest::SetUp();
```

Comment:
Below failures on the workflow could be related to this change. It looks like the Presto implementations were used for these tests. To fix that, we might need to avoid registering Presto functions in Spark's tests.

> [  FAILED  ] 4 tests, listed below:
[  FAILED  ] StringTest.md5
[  FAILED  ] StringTest.repeat
[  FAILED  ] StringTest.replace
[  FAILED  ] StringTest.sha1